neurolib/models/fhn/loadDefaultParams.py

```diff
@@ -38,7 +36,7 @@ class struct(object):
 
     # signal transmission speec between areas
     params.signalV = 20.0
-    params.K_gl = 250.0  # global coupling strength
+    params.K_gl = 0.6  # global coupling strength
```

Comment:
250 -> 0.6 is a pretty big difference 😄

Reply:
took the default values from `aln` before :) The main reason is because aLN works in `kHz`, hence the very low coupling strength (compared to `Hz` output, or anything really that is of order `1e0` to `1e2`.